lib/util/uint8array_utils.js

```diff
@@ -44,6 +44,26 @@ shaka.util.Uint8ArrayUtils.toBase64 = function(arr, padding) {
   return padding ? base64 : base64.replace(/=*$/, '');
 };
 
+/**
+ * Convert a Uint8Array to a base64 string. The output will be standard alphabet
+ * as opposed to "base64url" safe alphabet.
+ * @param {!Uint8Array} u8Arr
+ * @return {string}
+ * @export
+ */
+shaka.util.Uint8ArrayUtils.toStandardBase64 = function(u8Arr) {
+    const CHUNK_SIZE = 0x8000;
+    let index = 0;
+    const length = u8Arr.length;
+    let result = '';
+    let slice;
+    while (index < length) {
```

Comment:
We already have code exactly like this loop in StringUtils.fromCharCode().  Instead of this, try duplicating toBase64 above but without the replace() calls after btoa().

Then, toBase64 could be done in terms of toStandardBase64:

const base64 = toStandardBase64(bytes).replace(/\+/g, '-').replace(/\//g, '_');

Reply:
Thanks for the feedback!
Opened a new PR for this due to CLA:
https://github.com/google/shaka-player/pull/1932